src/applications/burials/components/IntroductionPage.jsx

```diff
@@ -4,23 +4,28 @@ import { focusElement } from 'platform/utilities/ui';
 import OMBInfo from '@department-of-veterans-affairs/component-library/OMBInfo';
 import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+import PropTypes, { shape } from 'prop-types';
```

Comment:
don't think we need to explicitly import shape here since its implicit with PropTypes